unfoldingWord/translationCore
# Upload Project Message Fix

#### This pull request addresses:
This PR address two things, the structure of the recent projects actions file and the order of actions dispatched for confirming and online action. 


#### How to test this pull request:
To test this PR try using the buttons in the project card and ensure working functionality.
CSV, and Upload Online were refactored into their own files so test those throughly.
Next make sure that the message displayed when a project is being uploading will show 'spinning tc icon'. This is the same message that shows when importing a project from online which should also be fixed now.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2272)
